Shu/README.md

```diff
@@ -0,0 +1,27 @@
+## Xinyuan Shu Student Developer Setup Onboarding
+
+1.Cloned fork to local computer using ssh key
+
+2.Created the draft pull request
+
+3.Installed VS Code into the local computer
+
+4.Created "test.py" program
+
+6.Installed pre-commit,worked through 'pre-commit introduction'
+
+7.Added the pre-commit config file to PR.
+
+8.Installed Docker, worked through 'The Carpentries Incubator module on Docker'
+
+9.Created Dockerfile,push dockerfile to docker hub
```

Comment:
Releasing an image to Docker Hub wasn't explicitly required, but that's cool that you did that! You should also document the steps that you needed to take to do so.